openslides_backend/action/motion/update_metadata.py

```diff
@@ -61,15 +62,31 @@ def set_recommendation_extension_reference_ids(
         self, instance: Dict[str, Any]
     ) -> None:
         recommendation_extension_reference_ids = []
+        orig_meeting_id = self.get_meeting_id(instance["id"])
         possible_rerids = RERIDS_PATTERN.findall(instance["recommendation_extension"])
         for fqid_str in possible_rerids:
             collection, id_ = fqid_str.split(KEYSEPARATOR)
+            if collection != "motion":
+                raise ActionException(f"Found {fqid_str} but only motion is allowed.")
             exists_res = self.datastore.exists(
                 collection=Collection(collection),
                 filter=FilterOperator("id", "=", int(id_)),
             )
             if exists_res.get("exists"):
+                meeting_id = self.get_meeting_id(id_)
+                if orig_meeting_id != meeting_id:
+                    raise ActionException(
+                        f"Meeting_ids of motion {instance['id']} and {id_} don't match."
+                    )
```

Comment:
You misunderstood me, the check itself is already handled by the `equal_fields`. Only a test for this was needed, so you should be able to remove this again with the test still working (apart from a different error message).

Reply:
This is not handled by equal_fields. I tested it. So my check is correct.

Reply:
You are right, but it should be. I fixed it in https://github.com/OpenSlides/openslides-backend/pull/354. Please rebase this branch and remove your manual check

Reply:
Rebased and removed the check. I needed to update the generic field handling too.